decompose/src/webMain/kotlin/com/arkivanov/decompose/router/stack/webhistory/DefaultWebHistoryController.kt

```diff
@@ -220,5 +246,7 @@ class DefaultWebHistoryController internal constructor(
         fun go(delta: Int)
         fun pushState(data: String, url: String?)
         fun replaceState(data: String, url: String?)
+        fun forward()
```

Comment:
Are these two methods actually used?

Reply:
`back` is used in the `doOnPopState` method, `forward` is not used.

Reply:
Shouldn't we use `go(-1)` instead of `back`?

Reply:
Yes, good point. That should be equivalent, per the documentation [here](https://developer.mozilla.org/en-US/docs/Web/API/History_API).